### DISCUSSION TITLE
I cant get rid of Tailwind error. no idea how to solve

### DISCUSSION THREAD

**Discussion by alexk400** [tailwindlabs/tailwindcss#12932] [General]
Was tinkering with svg icon.
Try to enter dynamic height and width using tailwindcss classes. Probably some quotes wrong.

1. cleared cache
2. deleted nextjs tailwind css  
3.  deleted .next folder several times
4. removed npm cache folder
5. cleared browser cache and tested in other browsers

It seems like tailwind css compiled by globals.css and stored it somewhere. Whether it is done by nextjs or tailwind.

Even changed globals.css to myglobals.css

./src/app/(frontend)/myglobals.css:1589:13
Parsing css source code failed
  1587 | }
  1588 | .min-w-\[\`\+defaultProps\.size\+\`\] {
> 1589 |   min-width: `+defaultProps.size+`;
       |              ^
  1590 | }
  1591 | .max-w-7xl {
  1592 |   max-width: 80rem;
  
  
  I removed all the code that caused this problem.
  
  I am stuck. Who is compiling and where its stored

**Comment by ShaunaB1** [comment]
To fix your Tailwind CSS issue, double-check your setup and syntax, especially for dynamic classes. Restart your development server to apply changes. If problems continue, ask the Tailwind community for help or look at the documentation. Sometimes, old CSS might be cached, so find and clear any caches. [Dog likes best](https://www.doglikesbest.com/)
